### PR TITLE
flake8 phase14

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://app.circleci.com/pipelines/github/demisto/content/17006/workflows/91d8b4b8-12e9-469e-8f53-1e2e73c798f9/jobs/59441/parallel-runs/1?filterBy=ALL

## Description
CheckDockerImageAvailable
ConvertFile
DBotTrainTextClassifierV2
GetShiftsPerUser

### CODE REVIEW

File: Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml [hunk 1]
```diff
@@ -182,7 +182,7 @@ tags:
 timeout: 12µs
 type: python
 subtype: python2
-dockerimage: demisto/dl:1.4
+dockerimage: demisto/dl:languages1.0
```
Comment (Itay4) [inline Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml:185]: @erezh31 @lperryp is this ok ?

Comment (lperryp) [reply Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml:185]: Actually dl:languages1.0 is much heavier than 1.4, it's better to leave it as it @Itay4 @jochman

**Comment by jochman** [comment]
Circle failed on known not latest docker image, can merge. @Itay4

**Comment by Itay4** [comment]
@jochman please resolve conflicts

**Comment by Itay4** [comment]
build fails on approved docker image validation
merging

**Pull request merged**